.gitignore

```diff
@@ -2,3 +2,4 @@ __pycache__/
 flickr_solved.py
 config.py
 *.py[oc]
+venvsec8/
```

Comment:
You created a virtual environment, which you don't have to, because your code does not use any library.